source/templates.tex

```diff
@@ -277,17 +277,16 @@
 \tcode{typename}
 followed by a
 \grammarterm{qualified-id}
-denotes the type in a non-type%
+denotes the type in a \grammarterm{parameter-declaration}.%
```

Comment:
We now have a footnote explaining what we mean by "non-type" attached to wording that doesn't say "non-type". That doesn't seem ideal. But in any case the disambiguation rule for `template<class T>` seems like a very strange place to be defining "non-type".